src/app/patient-table/patient-table.component.html

```diff
@@ -1,9 +1,9 @@
-<table
-    *ngIf="!!this.dataSource"
-    mat-table
-    [dataSource]="dataSource"
-    class="mat-elevation-z8"
->
+<div class="example-container mat-elevation-z8">
+    <div class="example-loading-shade">
```

Comment:
it could be a good idea to change the css class names to something meaningful, these are coming from the example... please change names of css classes